README.md

```diff
@@ -46,11 +46,12 @@ protected function __trigger()
     // If the ID isn't a number then it's a hash, so convert it to the entry ID
     if( isset($_POST['id']) && !is_numeric($_POST['id']) )
     {
-        require_once EXTENSIONS . '/hashid_field/lib/Hashids.php';
+        require_once EXTENSIONS . '/hashid_field/vendor/autoload.php';
+        useHashids;
```

Comment:
What is this line ?

Reply:
` require_once EXTENSIONS . '/hashid_field/lib/Hashids.php'; `doesn't exist there + doesn't work if i change the path:
This works:

` require_once EXTENSIONS '/hashid_field/vendor/hashids/hashids/lib/Hashids/HashGenerator.php';`	
` require_once EXTENSIONS . '/hashid_field/vendor/hashids/hashids/lib/Hashids/Hashids.php';`

or 
`require_once EXTENSIONS . '/hashid_field/vendor/autoload.php';  
useHashids;`

Reply:
I am not aware of what the 'useHashids;' bit is doing. I've never saw something like this and can not find documentation on that... 

I *think* the proper way would be `use Hashids;` (with a space ?) Or your could do

```php
require_once EXTENSIONS . '/hashid_field/vendor/autoload.php'; 
$hash = new \Hashids\Hashids( 'TheSaltForThisField' , 6 ); // note the extra \ after new
```